src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -3026,14 +3071,20 @@ private void getCallInfo(ref CORINFO_RESOLVED_TOKEN pResolvedToken, CORINFO_RESO
                     pResult.codePointerOrStubLookup.lookupKind.needsRuntimeLookup = false;
                     pResult.codePointerOrStubLookup.constLookup.accessType = InfoAccessType.IAT_PVALUE;
                     pResult.codePointerOrStubLookup.constLookup.addr = (void*)ObjectToHandle(
+#if READYTORUN
+                        _compilation.NodeFactory.InterfaceDispatchCell(targetMethod, pResolvedToken.token
+#else
                         _compilation.NodeFactory.InterfaceDispatchCell(targetMethod
 #if !SUPPORT_JIT
                         , _compilation.NameMangler.GetMangledMethodName(MethodBeingCompiled).ToString()
 #endif
+#endif // READYTORUN
                         ));
                 }
             }
-            else if ((flags & CORINFO_CALLINFO_FLAGS.CORINFO_CALLINFO_LDFTN) == 0
+#if !READYTORUN
+            else
```

Comment:
This is weird, I think I put the conditional compilation here by mistake when putting it on the above block.